htmlPlugin.js

```diff
@@ -0,0 +1,40 @@
+const { appInfo: { loki } } = require('./package.json');
+
+const resourceRootUrn = `urn:com:${loki.cloudCodeName}:${loki.appCodeName}:app:pages:${loki.pageCodeName}!`;
+/**
+ * @param {string} htmlstring An HTML string
+ * @param {string} filename A bundle file name to replace with "${loki.web.resourceUrl('[b]')}""
+ */
+function replacePathWithLokiResource(htmlstring, filename) {
+  const regexp = new RegExp(
+    `(=")(?:)[^=]*(?:)(${filename})(?:)[^=]*(")`,
+    'g',
+  );
+  return htmlstring.replace(
+    regexp,
+    `$1\${loki.web.resourceUrl('${resourceRootUrn}$2')}$3`,
+  );
+}
+const htmlPlugin = () => {
```

Comment:
@andrew-waller  @andrewhwaller  This basic vite plugin probably needs to be renamed and saved somewhere else. However, it finds the bundle files and replaces the path to the file with `${loki.web.resourceUrl('[BUNDLE_FILE_NAME]')}` so the page can run in any app that has permission to view it.

Reply:
We can look into abstracting this into an actual plugin; I need to see what the usual format for a Vite plugin is, but this should be ok for now.

Reply:
Sounds good. FWIW, this is a copy/paste from an example on the Vite site at [Plugin API](https://vitejs.dev/guide/api-plugin.html#transformindexhtml).